include/tvm/relay/dataflow_pattern.h

```diff
@@ -46,6 +48,29 @@ class DFPatternNode : public Object {
  */
 class DFPattern : public ObjectRef {
  public:
+  /*! \brief Syntatic Sugar for creating a CallPattern */
+  DFPattern operator()(const std::vector<DFPattern>& args);
+  /*! \brief Syntatic Sugar for creating a CallPattern with an "add" op */
```

Comment:
I feel that the syntactic sugar "+-*/" is not necessary since we have the "IsOp" one, and it may cause some confusion whether you're adding two patterns together or use the "add" op.

Reply:
Hmm. I did the +-*/ overloads in python because Relay's IR does the same thing at the python level, and I wanted to keep operation on the pattern API in line with Relay's API. I continued that here, trying to keep the C++ pattern API in line with the Python Pattern API. So...for consistency I'd argue to leave it as it is.

Reply:
I see. sure.